plugins/woocommerce/src/Admin/API/Init.php

```diff
@@ -49,6 +49,9 @@ public function __construct() {
 
 		// Add currency symbol to orders endpoint response.
 		add_filter( 'woocommerce_rest_prepare_shop_order_object', array( __CLASS__, 'add_currency_symbol_to_order_response' ) );
+
+		include_once __DIR__ . '/../../../includes/admin/class-wc-admin-upload-downloadable-product.php';
```

Comment:
Would we potentially want to use `WC_ABSPATH . 'includes....` here? instead of the `/../../..`?

Reply:
Sure, I get lost in all those constants :)